.circleci/config.yml

```diff
@@ -2,8 +2,8 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.13
-    working_directory: /go/src/github.com/deislabs/smi-sdk-go
+      - image: circleci/golang:1.14
```

Comment:
Can we break the go change out into a different PR please? I just don't want to conflate things.